packages/web-app/src/components/appli/Dashboard/index.jsx

```diff
@@ -0,0 +1,186 @@
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { Box, Typography } from '@material-ui/core';
+import Skeleton from '@material-ui/lab/Skeleton';
+import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import AssessmentIcon from '@material-ui/icons/Assessment';
+import { fetchStatisticsCountry } from '../../../actions/Country/GetStatisticsCountry';
+import { fetchStatisticsMassif } from '../../../actions/Massif/GetStatisticsMassif';
+import SpecificsCaves from './components/SpecificsCaves';
+import CavesData from './components/CavesData/index';
+import CavesStatistics from './components/CavesStatistics';
+import Alert from '../../common/Alert';
+
+const Title = withStyles(() => ({
+  root: {
+    paddingLeft: '15px',
+    verticalAlign: 'middle',
+    display: 'inline-flex'
+  }
+}))(Typography);
+
+const TitleBox = withStyles(
+  theme => ({
+    root: {
+      backgroundColor: theme.palette.primary.light,
+      color: '#FFFFFF',
+      padding: '15px 30px',
+      marginBottom: '20px'
+    }
+  }),
+  { withTheme: true }
+)(Box);
+
+const DataBox = withStyles(() => ({
+  root: {
+    margin: '0px 2%',
+    marginTop: '30px'
+  }
+}))(Box);
+
+const DashboardBox = withStyles(
+  theme => ({
+    root: {
+      backgroundColor: theme.palette.backgroundColor,
+      margin: '10px 5%'
+    }
+  }),
+  { withTheme: true }
+)(Box);
+
+const Dashboard = ({ countryId, massifId }) => {
```

Comment:
Can we name this component (and the folder) differently?

`Dashboard` seems a bit too generic

Reply:
Since we are using this component to display the dashboard on the massif and country pages, we wanted a global name that would work for both cases. 
Is 'StatisticsData' a more suitable name ?

Reply:
StatisticsDataDashboard !